src/current/_includes/v23.1/essential-metrics.md

```diff
@@ -0,0 +1,178 @@
+These essential CockroachDB metrics enable you to build custom dashboards with tools, such as: 
+{% if include.deployment == 'self-hosted' %}
+* [Grafana](monitor-cockroachdb-with-prometheus.html#step-5-visualize-metrics-in-grafana)
+* [Datadog Integration](datadog.html) - The [**Datadog Integration Metric Name**](https://docs.datadoghq.com/integrations/cockroachdb/?tab=host#metrics) column lists the corresponding Datadog metric which requires the `cockroachdb.` prefix. A given metric is not available using the Datadog Integration if the **Datadog Integration Metric Name** column is blank (such as for the `sys.cpu.host.combined.percent-normalized` metric).
+{% elsif include.deployment == 'dedicated' %}
+* [Datadog integration](../cockroachcloud/tools-page.html#monitor-cockroachdb-dedicated-with-datadog) - The [**Datadog Integration Metric Name**](https://docs.datadoghq.com/integrations/cockroachdb_dedicated/#metrics) column lists the corresponding Datadog metric which requires the `crdb_dedicated.` prefix. A given metric is not available using the Datadog Integration if the **Datadog Integration Metric Name** column is not shown (such as for Row-Level TTL, Backup and Restore, and Table Statistics categories) or is blank (such as for the `sys.cpu.host.combined.percent-normalized` metric).
+* [Metrics export](../cockroachcloud/export-metrics.html)
+{% endif %}
+  
+The metrics are important to visualize in a custom dashboard. The **Usage** column explains how to make both practical and actionable use of the metric in a production deployment.
+
+## Platform
+
+|<div style="width:225px">CockroachDB Metric Name<div> | {% if include.deployment == 'self-hosted' %}<div style="width:225px">[Datadog Integration Metric Name](https://docs.datadoghq.com/integrations/cockroachdb/?tab=host#metrics)<br>(add `cockroachdb.` prefix)<div> |{% elsif include.deployment == 'dedicated' %}<div style="width:225px">[Datadog Integration Metric Name](https://docs.datadoghq.com/integrations/cockroachdb_dedicated/#metrics)<br>(add `crdb_dedicated.` prefix)<div> |{% endif %} Description | Usage |
```

Comment:
Is it safe to assume all of these metrics/description were pulled in from the main metrics docs and have already been reviewed?

Reply:
The descriptions were not necessarily pulled from the metrics docs but have been reviewed by the field team. I double-checked them against the tooltip descriptions in the [Custom Chart Debug Page] https://www.cockroachlabs.com/docs/v23.1/ui-custom-chart-debug-page).